nengo_loihi/simulator.py

```diff
@@ -509,13 +511,18 @@ def loihi_precomputed_host_only(steps):
             assert host_pre is None
 
             if host is not None:
+                self.timers["snips"] = 0
 
                 def loihi_bidirectional_with_host(steps):
                     loihi.run_steps(steps, blocking=False)
+                    time0 = timeit.default_timer()
                     for _ in range(steps):
                         host.step()
                         self._host2chip(loihi)
                         self._chip2host(loihi)
+                    time1 = timeit.default_timer()
+                    self.timers["snips"] += time1 - time0
```

Comment:
The variable names `time0` and `time1` can be renamed to something more descriptive e.g. `time_snips_start` and `time_snips_end`. Also, similar to `self.timers["steps"]`, `time1` can be eliminated by `self.timers["snips"] += timeit.default_timer() - time0` unless there is other plans to use `time1`